app/users/dashboard/finance/budgets/[id]/page.tsx

```
@@ -159,268 +158,387 @@ export default function BudgetDetailPage({ params }: BudgetDetailPageProps) {
+                <div className="flex items-center gap-3 rounded-lg border border-green-200 bg-green-50 p-4 dark:border-green-900/20 dark:bg-green-900/10">
+                  <CheckCircle className="h-5 w-5 shrink-0 text-green-600 dark:text-green-400" />
+                  <div>
+                    <p className="font-semibold text-green-900 dark:text-green-100">
+                      Healthy
+                    </p>
+                    <p className="text-sm text-green-800 dark:text-green-200">
+                      {budget.percentageUsed.toFixed(1)}% of budget used
+                    </p>
+                  </div>
+                </div>
+              )}
```

ESLint

[error] Nested ternary expression should be parenthesized.

(unicorn/no-nested-ternary)